account_chart_update_multilang/wizards/wizard_chart_update.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from odoo import models
+
+
+class WizardUpdateChartsAccount(models.TransientModel):
+    _inherit = "wizard.update.charts.accounts"
+
+    def diff_translate_fields(self, template, real):
+        """Find differences by comparing the translations of the fields."""
+        res = {}
+        langs = self.env["res.lang"].search([])
+        for key, _ in template._fields.items():
+            if not template._fields[key].translate:
+                continue
+            for lang in langs:
+                template_trans = getattr(template.with_context(lang=lang.code), key)
+                real_trans = getattr(real.with_context(lang=lang.code), key)
+                if template_trans != real_trans:
+                    res[key] = template[key]
+                    continue
+        return res
+
+    def diff_fields(self, template, real):
+        """Allow the comparison of template translations with real
+        translations."""
+        res = super().diff_fields(template, real)
+        res.update(self.diff_translate_fields(template, real))
+        return res
+
+    def _update_taxes(self):
+        """Update the taxes with their template translations."""
+        res = super()._update_taxes()
+        for wiz_tax in self.tax_ids.filtered(lambda x: x.type != "new"):
```

Comment:
suggestion: don't double-iterate. In general terms, if you're gonna iterate over a recordset, filtering it just double-iterates it. In this particular case it won't affect performance very much. But yet again, it's a good rule of thumb:

```suggestion
        for wiz_tax in self.tax_ids:
            if wiz_tax.type != "new":
                continue
```

This way you iterate only once.

Reply:
I see the point. But if i want to exclude "new" wiz_tax then it should be:

`if wiz_tax.type == "new"`

Reply:
Ah of course! Good catch